pom.xml

```diff
@@ -135,8 +135,26 @@
           <goals>deploy</goals>
         </configuration>
       </plugin>
+      <!-- TODO: remove when library parent POM enables FindBugs by default -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <effort>Max</effort>
+          <!-- TODO: issues need to be fixed first-->
```

Comment:
Is it worth adding `<threshold>Low</threshold>` (or whatever the correct config), or is that mostly just useful for catching serialization issues?

Reply:
I have started from that, but now it generates a long list of garbage issues like serialVersionID & Co. I think we need to fix them first.